### DISCUSSION TITLE
How do I make `v2raya` use the geoip.dat and geosite.dat from the v2ray-core folder.

### DISCUSSION THREAD

**Discussion by siyabuilds** [v2rayA/v2rayA#1480] [Q&A]
Hello. I am used to doing things in Linux which kinda normalized keeping things easy honnestly but now I am using windows and whenever I try running v2raya from the Command Prompt I keep running into the same error:

```bash
C:\Users\Smerfor\Downloads\v2ray-cli>v2raya_windows --lite --v2ray-bin v2ray.exe
2024/08/19 11:45:29.802 [A] [main.go:27]  downloading missing geoip.dat and geosite.dat
2024/08/19 11:45:30.141 [I] [geoip.go:16]  Downloading https://github.com/v2fly/geoip/releases/latest/download/geoip.dat to C:\Users\Smerfor\AppData\Roaming\v2raya\geoip.dat.new
2024/08/19 11:47:00.227 [W] [pre.go:190]  UpdateLocalGeoIP: context deadline exceeded (Client.Timeout or context cancellation while reading body)
2024/08/19 11:47:00.372 [E] [main.go:27]  UpdateLocalGeoIP: context deadline exceeded (Client.Timeout or context cancellation while reading body)

C:\Users\Smerfor\Downloads\v2ray-cli>
```

Please note that every file is kept in the same directory. I tried copying the geofiles manually to the AppData folder and that still didn't work. If anyone has a solution or fix I would greatly appreciate the insight.

**N.B**: I tried even downloading the files using my browser and they download perfectly which means the problem isn't related to network.

**Comment by iopq** [comment]
the solution is to put the files in the folder the application is looking for them

I don't use Windows, but you basically didn't put in the right place
